csm/conf/configure.py

```diff
@@ -256,3 +267,42 @@ def get_component_list_from_features_endpoints():
         except Exception as e_:
             Log.error(f"Error in storing unsupported features: {e_}")
             raise CsmSetupError(f"Error in storing unsupported features: {e_}")
+
+    async def _create_cluster_admin(self):
+        # TODO confstore keys can be changed.
```

Comment:
as discussed please move this logic to usermanager class

Reply:
Modifications done